### PR TITLE
Add section to FAQ explaining Northstar

### PR DESCRIPTION
**Pull request by GeckoEidechse**
When someone asked on our Discord how Northstar actually works it made me realise that we never added any explanation in the wiki so this little FAQ section tries to close that gap ^^

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
Quick feedback I got via Discord message

> what is a mod?
> what does hook mean?
> launcher doesnt really hook into scripts
> what is a `server.dll` and why does it ship with a game client ( might be worth omitting the client after Titanfall2 )
> "re-uses library to host servers" hmmmm
> don't think the whole security ramble needs to be that long
> "adding additional gamemodes for new custom gamemodes" hmmmm
> 
> just some thoughts

File: docs/Wiki/faq.md [hunk 1]
```diff
@@ -6,6 +6,17 @@ description: Frequently asked questions
 
 ## **If you have any issues please go to [the troubleshooting page.](installing-northstar/troubleshooting.md)**
 
+### Q: What is Northstar and how does it work? <a href="#what-is-northstar" id="what-is-northstar"></a>
+
+A: Northstar is a mod for Titanfall2 that allows you to run your own gameservers with custom gamemodes, weapons, maps, and skins.
+
+It consists out of the launcher that hooks into the game and scripts that replicate vanilla functionality and extend them.
+
+The launcher re-purposes the existing `server.dll` that is shipped  with the Titanfall2 client for running single player (sp is essentially just a local server) and then re-uses library to host servers that other players can join.
```
Comment (JMM889901) [inline docs/Wiki/faq.md:15]: > re-uses library 

should this be "re-uses this library"
should probably be written
```
The launcher re-purposes the existing `server.dll` that is shipped with the Titanfall2 client for running single-player (which is essentially just a local server) to host servers that other players can join.
```

File: docs/Wiki/faq.md [hunk 2]
```diff
@@ -6,6 +6,17 @@ description: Frequently asked questions
...
+From there a bunch of logic is added to ensure some levels of security as the vanilla client fully trusts server (which is fine for vanilla where servers are run by Respawn but is not enough for Northstar cause anyone can host a server including malicious entities) as well as additional features and functionality.
+
+The game scripts replicate vanilla functionality to mimic the gamemodes that can be played in vanilla as well as adding additional gamemodes for new custom gamemodes.
```
Comment (JMM889901) [inline docs/Wiki/faq.md:18]: > adding additional gamemodes for new custom gamemodes.

i think you rewrote this and forgot to remove the previous one

**Review by JMM889901 — CHANGES_REQUESTED** [review]

File: docs/Wiki/faq.md (hunk 2 above)
Comment (GeckoEidechse) [reply docs/Wiki/faq.md]: Fixed via 75082fac5e7fabaa4996944b058044ffd2a84c33

Thanks ^^

**Review by JMM889901 — APPROVED** [review]

**Pull request merged**